plugin/kubernetes/object/endpoint.go

```diff
@@ -27,10 +27,11 @@ type EndpointSubset struct {
 
 // EndpointAddress is a tuple that describes single IP address.
 type EndpointAddress struct {
-	IP            string
-	Hostname      string
-	NodeName      string
-	TargetRefName string
+	IP                 string
+	Hostname           string
+	NodeName           string
+	TargetRefName      string
+	TargetRefNamespace string
```

Comment:
this is need cheap....

Reply:
Yes I know. I cover this in the description.